includes/misc-functions.php

```diff
@@ -2477,4 +2477,26 @@ function give_display_donation_receipt( $args ) {
 	give_get_template_part( 'shortcode', 'receipt' );
 	
 	return ob_get_clean();
+}
+
+/**
+ * This function is used to prepare url with query strings on top of donation history url.
+ *
+ * @param bool|array $args List of query strings to be added.
+ *
+ * @since 2.4.1
+ *
+ * @return string
+ */
+function give_prepare_donation_history_url( $args = false ) {
```

Comment:
@mehul0810 I did not find this function useful, either revert it  or add `only for internal` use tag

Reply:
@ravinderk I've reverted that function and updated in both the PR and tested the functionality again to ensure it is working fine.